internal/updater/html/parsing.go

```diff
@@ -0,0 +1,98 @@
+package htmlutils
+
+import (
+	"container/list"
+	"errors"
+	"fmt"
+
+	"golang.org/x/net/html"
+)
+
+var (
+	ErrAttrNotFound = errors.New("matching attribute not found")
+)
+
+func GetText(n *html.Node) string {
+	return n.FirstChild.Data
+}
+
+func GetAttr(n *html.Node, key string) (string, error) {
+	for _, attr := range n.Attr {
+		if attr.Key == key {
+			return attr.Val, nil
+		}
+	}
+
+	return "", ErrAttrNotFound
+}
+
+func CheckAttrMatch(n *html.Node, attrKey string, checkValue string) bool {
+	attrValue, err := GetAttr(n, attrKey)
+	return err == nil && attrValue == checkValue
+}
+
+func CheckID(n *html.Node, idValue string) bool {
```

Comment:
nit having named returns help for documentation

```suggestion
func CheckID(n *html.Node, idValue string) (match bool) {
```